x-pack/legacy/plugins/infra/public/pages/logs/analysis/page_results_content.tsx

```diff
@@ -150,29 +133,8 @@ export const AnalysisResultsContent = ({
           <EuiPage>
             <EuiPanel paddingSize="l">
               <EuiFlexGroup justifyContent="spaceBetween">
-                <EuiFlexItem grow={7}>
-                  <EuiFlexGroup alignItems="center">
-                    <EuiFlexItem grow={false}>
-                      {anomaliesDetected !== null ? (
-                        <span>
-                          <FormattedMessage
-                            id="xpack.infra.logs.analysis.anomaliesDetectedText"
-                            defaultMessage="Detected {formattedNumber} {number, plural, one {anomaly} other {anomalies}}"
-                            values={{
-                              formattedNumber: (
-                                <EuiBadge color={anomaliesDetected === 0 ? 'default' : 'warning'}>
-                                  {anomaliesDetected}
-                                </EuiBadge>
-                              ),
-                              number: anomaliesDetected,
-                            }}
-                          />
-                        </span>
-                      ) : null}
-                    </EuiFlexItem>
-                  </EuiFlexGroup>
-                </EuiFlexItem>
-                <EuiFlexItem>
+                <EuiFlexItem></EuiFlexItem>
```

Comment:
Is this empty element intentional?

Reply:
It is in this implementation. In the followup PR I'll amend this so we justify / align things a little differently.